modules/jira-configuring-acs.adoc

```diff
@@ -10,15 +10,24 @@ Create a new integration in {product-title} by using the Jira server URL and use
 .Procedure
 . In the {product-title-short} portal, go to *Platform Configuration* -> *Integrations*.
 . Scroll down to the *Notifier Integrations* section and select *Jira Software*.
-. Click *New Integration*.
-. Enter a name for *Integration Name*.
-. Enter the user credentials in the *Username* and *Password or API Token* boxes.
-. For *Issue Type*, enter a valid link:https://confluence.atlassian.com/adminjiracloud/issue-types-844500742.html[Jira Issue Type], for example *Task*, *Sub-task*, or *Bug*.
-. Enter the Jira server URL in the *Jira URL* box.
-. Enter the key of the project in which you want to create issues in the *Default Project* box.
-. Use the *Annotation Key For Project* box to create issues in different Jira projects.
+. Click *New integration*.
+. Enter a name for *Integration name*.
+. Enter the user credentials in the *Username* and *Password or API token* fields.
+. For *Issue type*, enter a valid link:https://confluence.atlassian.com/adminjiracloud/issue-types-844500742.html[Jira Issue Type], for example *Task*, *Sub-task*, or *Bug*.
+. Enter the Jira server URL in the *Jira URL* field.
+. Enter the key of the project in which you want to create issues in the *Default project* field.
+. Optional: Use the *Annotation key for project* field to create issues in different Jira projects. You can use annotations to dynamically create issues. To do this:
+
+.. Add an annotation similar to the following example in your namespace or deployment YAML file, where `jira/project-key` is the annotation key that you specify in your Jira integration. You can create an annotation for the deployment or the namespace.
++
+[source,yaml,subs="+quotes"]
+----
+annotations:
+  jira/project-key: _<jira_project_key>_
+----
```

Comment:
```suggestion
[source,yaml,subs="+quotes"]
----
annotations:
# ...
  jira/project-key: <jira_project_key>
# ...
```

Reply:
As per this [guideline](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#signs-and-symbols-in-code-blocks).